src/Components/Facility/PatientNotesSlideover.tsx

```diff
@@ -37,6 +36,11 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
   const { facilityId, patientId, consultationId, setShowPatientNotesPopup } =
     props;
 
+  const localStorageKey = `patientNotesNoteField_${patientId}`;
```

Comment:
This is specific to a patient, not specific to a consultation.
```suggestion
  const localStorageKey = `patientNotesNoteField_${consultationId}`;
```